pyfr/solvers/navstokes/kernels/bccent.mako

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+<%inherit file='base'/>
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%include file='pyfr.solvers.navstokes.kernels.bcs.${bctype}'/>
+<%include file='pyfr.solvers.euler.kernels.entropy'/>
```

Comment:
Should we be pulling in NS here?  As in `bent.mako` we pull in `pyfr.solvers.navstokes.kernels.entropy`.

Reply:
NS uses the same entropy function as Euler

Reply:
That makes sense, although we have `<%include file='pyfr.solvers.navstokes.kernels.entropy'/>` in `pyfr/solvers/navstokes/kernels/bent.mako`

Reply:
Good catch, that file isn't used anymore. Deleted now